scylla/datadog_checks/scylla/scylla.py

```diff
@@ -0,0 +1,63 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from six.moves.urllib.parse import urlparse
+
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+from datadog_checks.base.errors import CheckException
+
+from .metrics import ADDITIONAL_METRICS_MAP, INSTANCE_DEFAULT_METRICS
+
+
+class ScyllaCheck(OpenMetricsBaseCheck):
+    """
+    Collect Scylla metrics from Prometheus endpoint
+    """
+
+    DEFAULT_METRIC_LIMIT = 0
+
+    def __init__(self, name, init_config, instances):
+
+        instance = instances[0]
+
+        endpoint = instance.get('prometheus_url')
+        if endpoint is None:
+            # needed to guard for Python2
+            # empty endpoint will raise CheckException at init, but Py2 urlparse will raise AttributeError instead
+            raise CheckException("Unable to find prometheus URL in config file.")
```

Comment:
I thought about that - but chose to use `CheckException` since that's what the base class will do if if finds an empty url value.

On the other hand, could also just make this change and remove the comments about guarding for Python2 ..